### PR TITLE
Put Nov data release notes in place to be linked in upon release

### PR DESCRIPTION
**Pull request by timtheisen**

### CODE REVIEW

**Comment by brianhlin** [comment]
One downside of this method that these pages will be indexed by search engines. Probably not a big deal but it's something to think about

File: docs/release/3.3/release-3-3-30-2.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+This release contains:
+
+-   [VO Package v76](https://github.com/opensciencegrid/osg-vo-config/releases/tag/release-76)
+    -   Drop redundant geant4-lcgadmin objects
+    -   Add missing SNO+ VOMS servers
+    -   Add notice to voms-mapfile-default
+
+These [JIRA tickets](https://jira.opensciencegrid.org/issues/?jql=project%20%3D%20SOFTWARE%20AND%20fixVersion%20%3D%203.3.30-2%20ORDER%20BY%20priority%20DESC%2C%20key%20DESC) were addressed in this release.
+
+Detailed changes are below. All of the documentation can be found in the [Release3](https://twiki.grid.iu.edu/bin/view/Documentation/Release3/) area of the TWiki.
```
Comment (matyasselmeci) [inline docs/release/3.3/release-3-3-30-2.md]: This should point to the new docs area instead.

File: docs/release/3.4/release-3-4-5-2.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+This release contains:
+
+-   [VO Package v76](https://github.com/opensciencegrid/osg-vo-config/releases/tag/release-76)
+    -   Drop redundant geant4-lcgadmin objects
+    -   Add missing SNO+ VOMS servers
+    -   Add notice to voms-mapfile-default
+
+These [JIRA tickets](https://jira.opensciencegrid.org/issues/?jql=project%20%3D%20SOFTWARE%20AND%20fixVersion%20%3D%203.4.5-2%20ORDER%20BY%20priority%20DESC%2C%20key%20DESC) were addressed in this release.
+
+Detailed changes are below. All of the documentation can be found in the [Release3](https://twiki.grid.iu.edu/bin/view/Documentation/Release3/) area of the TWiki.
```
Comment (matyasselmeci) [inline docs/release/3.4/release-3-4-5-2.md]: This should point to the new docs area.

File: docs/release/3.4/release-3-4-5-2.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+Detailed changes in this release
+--------------------------------
+
+### Packages
+
+We added or updated the following packages to the production OSG yum repository. Note that in some cases, there are multiple RPMs for each package. You can click on any given package to see the set of RPMs or see the complete list below.
+
+#### Enterprise Linux 6
+
+-   [vo-client-76-1.osg33.el6](https://koji.chtc.wisc.edu/koji/search?match=glob&type=build&terms=vo-client-76-1.osg33.el6)
```
Comment (matyasselmeci) [inline docs/release/3.4/release-3-4-5-2.md]: Should point to the osg34 build instead.

File: docs/release/3.4/release-3-4-5-2.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+#### Enterprise Linux 7
+
+-   [vo-client-76-1.osg33.el7](https://koji.chtc.wisc.edu/koji/search?match=glob&type=build&terms=vo-client-76-1.osg33.el7)
```
Comment (matyasselmeci) [inline docs/release/3.4/release-3-4-5-2.md]: Should point to the osg34 build instead

File: docs/release/3.4/release-3-4-5-2.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+### RPMs
+
+If you wish to manually update your system, you can run yum update against the following packages:
+
+    osg-gums-config vo-client vo-client-edgmkgridmap vo-client-lcmaps-voms
+
+If you wish to only update the RPMs that changed, the set of RPMs is:
+
+#### Enterprise Linux 6
```
Comment (matyasselmeci) [inline docs/release/3.4/release-3-4-5-2.md]: Package lists are for 3.3 - change to 3.4

**Comment by timtheisen** [comment]
Yes, but the advantage is the it is searchable on release day. For regular releases, that hang out for weeks, I would want them on doc-itb. Developers need to be able to record known issues before release.

**Pull request merged**